code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -40,18 +40,9 @@
 /obj/item/weapon/legcuffs/bola/throw_impact(atom/hit_atom)
 	if(..() || !ishuman(hit_atom))//if it gets caught or the target can't be cuffed,
 		return//abort
-	var/mob/living/carbon/human/H = hit_atom
-	/*
-	var/num_legs
+	cacht(hit_atom)
```

Comment:
cacht? I'm thinking you mean catch, although proc names should probably be more descriptive